javascript/package.json

```diff
@@ -31,7 +31,6 @@
     "@cucumber/gherkin": "22.0.0",
     "@cucumber/gherkin-streams": "^5.0.0",
     "@cucumber/gherkin-utils": "^7.0.0",
-    "@cucumber/message-streams": "^4.0.0",
```

Comment:
Had to remove this, otherwise:

```
mpkorstanje@Nyx:~/Projects/cucumber/html-formatter/javascript$ make prepare
make: *** No rule to make target 'prepare'.  Stop.
mpkorstanje@Nyx:~/Projects/cucumber/html-formatter/javascript$ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @cucumber/gherkin-streams@5.0.0
npm ERR! Found: @cucumber/message-streams@4.0.0
npm ERR! node_modules/@cucumber/message-streams
npm ERR!   dev @cucumber/message-streams@"^4.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @cucumber/message-streams@"^3.0.0" from @cucumber/gherkin-streams@5.0.0
npm ERR! node_modules/@cucumber/gherkin-streams
npm ERR!   dev @cucumber/gherkin-streams@"^5.0.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @cucumber/message-streams@3.0.0
npm ERR! node_modules/@cucumber/message-streams
npm ERR!   peer @cucumber/message-streams@"^3.0.0" from @cucumber/gherkin-streams@5.0.0
npm ERR!   node_modules/@cucumber/gherkin-streams
npm ERR!     dev @cucumber/gherkin-streams@"^5.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/mpkorstanje/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/mpkorstanje/.npm/_logs/2022-04-03T17_54_55_095Z-debug-0.log
```
Dunno if this makes sense.